dockerfiles/django/start-dev.sh

```diff
@@ -4,4 +4,4 @@
 
 python -m manage migrate --noinput
 
-python -m manage tailwind --skip-checks runserver 0.0.0.0:8000
+python -m manage runserver --skip-checks 0.0.0.0:8000
```

Comment:
I think this should be reverted, as the `runserver` command provided by django-tailwind-cli needs to be used so styles are automatically generated.